engine/consensus/dkg/reactor_engine.go

```diff
@@ -161,33 +229,56 @@ func (e *ReactorEngine) EpochSetupPhaseStarted(currentEpochCounter uint64, first
 	e.registerPhaseTransition(curDKGInfo.phase3FinalView, dkgmodule.Phase3, e.end(nextEpochCounter))
 }
 
-// EpochCommittedPhaseStarted handles the EpochCommittedPhaseStarted protocol event. It
-// compares the key vector locally produced by Consensus nodes against the FlowDKG smart contract
-// key vectors. If the keys don't match a log statement will be invoked. In the happy case the locally
-// produced key should match, if the keys do not match the node will have a invalid random beacon key.
-func (e *ReactorEngine) EpochCommittedPhaseStarted(currentEpochCounter uint64, first *flow.Header) {
+// handleEpochCommittedPhaseStarted checks that the local DKG completed and
+// that our locally computed key share is consistent with the canonical key vector.
+//
+// If our key is inconsistent, we log a warning message.
+// CAUTION: we will still sign using the inconsistent key, to be fixed with Crypto
+// v2 by falling back to staking signatures in this case.
+//
+func (e *ReactorEngine) handleEpochCommittedPhaseStarted(currentEpochCounter uint64) {
+
+	// the epoch counter the DKG just completed is for
+	dkgEpochCounter := currentEpochCounter + 1
```

Comment:
better to create a logger, include both epoch counter value in the fields, and use the logger to log success case or failure case

Reply:
https://github.com/onflow/flow-go/pull/1721/commits/b5e69b4507024026c63471c93ecd41229b4f5365